zowe-cli-id-federation-plugin/package.json

```diff
@@ -0,0 +1,136 @@
+{
+  "name": "@zowe/id-federation-for-zowe-cli",
+  "version": "2.6.0",
+  "description": "An Identity Federation plugin for Zowe CLI",
+  "homepage": "https://github.com/zowe/api-layer/zowe-cli-id-federation-plugin#readme",
+  "bugs": {
+    "url": "https://github.com/zowe/api-layer/issues"
+  },
+  "keywords": [
+    "zowe",
+    "cli",
+    "id-federation",
+    "plugin"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/zowe/api-layer.git"
+  },
+  "main": "lib/index.js",
+  "files": [
+    "lib"
+  ],
+  "publishConfig": {
+    "registry": "https://zowe.jfrog.io/zowe/api/npm/npm-local-release/"
+  },
+  "scripts": {
+    "build": "tsc --pretty && npm run checkTestsCompile && madge -c lib",
+    "checkTestsCompile": "echo \"Checking that test source compiles...\" && tsc --project __tests__/test-tsconfig.json --noEmit ",
+    "prebuild": "npm run clean && npm run lint && echo Using TypeScript && tsc --version",
+    "clean": "rimraf lib",
+    "watch": "tsc --pretty --watch",
+    "prepublishOnly": "npm run build",
+    "lint": "eslint \"src/**/*.ts\" \"**/__tests__/**/*.ts\"",
+    "lint:src": "eslint \"src/**/*.ts\" --ignore-pattern \"**/__tests__/**/*.ts\"",
+    "lint:tests": "eslint \"**/__tests__/**/*.ts\"",
+    "test": "npm run test:unit && npm run test:integration && npm run test:system",
+    "test:system": "env-cmd __tests__/__resources__/env/system.env jest .*/__system__/.* --coverage false",
+    "test:integration": "env-cmd __tests__/__resources__/env/integration.env jest .*/__integration__/.* --coverage false",
+    "test:unit": "env-cmd __tests__/__resources__/env/unit.env jest --coverage --testPathIgnorePatterns \".*/__system__|__integration__/.*\"",
+    "installPlugin": "npm install && npm run clean && npm run build && zowe plugins install .",
+    "typedoc": "typedoc --out ./docs/typedoc/ ./src/",
+    "prepare": "cd .. && husky install"
+  },
+  "imperative": {
+    "configurationModule": "lib/imperative.js"
+  },
+  "dependencies": {
+    "@zowe/core-for-zowe-sdk": "7.0.0",
+    "@zowe/zosmf-for-zowe-sdk": "7.0.0"
+  },
+  "devDependencies": {
+    "@types/fs-extra": "8.0.0",
+    "@types/jest": "27.0.1",
+    "@types/node": "8.10.66",
+    "@typescript-eslint/eslint-plugin": "4.29.0",
+    "@typescript-eslint/parser": "4.29.0",
+    "@zowe/cli": "7.10.4",
+    "@zowe/cli-test-utils": "7.10.4",
+    "@zowe/imperative": "5.8.3",
+    "env-cmd": "8.0.2",
+    "eslint": "7.32.0",
+    "eslint-plugin-jest": "24.4.0",
+    "eslint-plugin-license-header": "0.4.0",
+    "eslint-plugin-unused-imports": "1.1.2",
+    "fs-extra": "8.1.0",
+    "husky": "7.0.0",
+    "jest": "29.4.2",
+    "jest-cli": "29.4.2",
+    "jest-environment-node": "29.4.2",
+    "jest-environment-node-debug": "2.0.0",
+    "jest-html-reporter": "3.7.0",
+    "jest-junit": "6.3.0",
+    "jest-stare": "^2.5.0",
```

Comment:
could you please use the exact version, without ^

Reply:
Somehow this one hided from me...